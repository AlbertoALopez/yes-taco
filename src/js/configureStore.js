import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
import thunk from 'redux-thunk';

import cloudDetectionReducer from './services/cloudDetection/reducers.js';
import imageUploadReducer from './services/imageUpload/reducers.js';

export default function configureStore() {

  const reducer = combineReducers({
    cloudDetectionReducer,
    imageUploadReducer,
  });

  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

  const store = createStore(reducer, composeEnhancers(applyMiddleware(thunk)));

  return store;
}
