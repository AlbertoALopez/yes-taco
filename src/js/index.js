import React from 'react';
import ReactDOM from 'react-dom';

// import App from './components/App';
import Main from './scenes/main';

import '../style/index.scss';

export default ReactDOM.render(
  <Main />,
  document.getElementById('root') || document.createElement('div'),
);
