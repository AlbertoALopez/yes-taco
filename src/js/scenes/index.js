import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { Route } from 'react-router-dom';
import { AppContainer } from 'react-hot-loader';

import Main from './main';

const Routes = () => (
  <div>
    <Route path="/" component={Main} />
  </div>
);

const App = () => (
  <AppContainer>
    <BrowserRouter>
      <Routes />
    </BrowserRouter>
  </AppContainer>
);

export default App;
