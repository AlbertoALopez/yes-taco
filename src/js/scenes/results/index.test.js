import React from 'react';
import { Results } from './index.js';

test('Loading renders correctly', () => {
  const props = {
    imageLabels: [{ description: 'taco', score: 100 }],
    file: {},
    history: {
      push: jest.fn(),
    },
  };

  const wrapper = shallow(<Results {...props} />);

  expect(wrapper).toMatchSnapshot();
});
