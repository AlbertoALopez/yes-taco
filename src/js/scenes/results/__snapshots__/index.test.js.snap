// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<div
  className="results-container"
>
  <Col
    xs={12}
  >
    <Row
      center="xs"
    >
      <Dialog>
        <div>
          <h3>
            Yes Taco!
          </h3>
        </div>
      </Dialog>
    </Row>
  </Col>
</div>
`;
